AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -1137,6 +1129,23 @@ - (BOOL)openURL:(NSURL *)url {
   return YES;
 }
 
+- (void)processDistributionGroupId:(NSString *)queryDistributionGroupId {
+
+  // Storing the distribution group ID to storage.
+  [MS_USER_DEFAULTS setObject:queryDistributionGroupId forKey:kMSDistributionGroupIdKey];
+
+  // Update distribution group ID which is added to logs.
+  [self.distributeInfoTracker updateDistributionGroupId:queryDistributionGroupId];
+
+  // Only if we have managed to retrieve the Distribution group ID we should update the distribution session count.
+  NSString *latestSessionId = [[MSSessionContext sharedInstance] sessionIdAt:[NSDate date]];
```

Comment:
Why do we need to get it based on current time instead of just `sessionId`?
```suggestion
  NSString *latestSessionId = [[MSSessionContext sharedInstance] sessionId];
```